NetKet/Machine/activations.hpp

```diff
@@ -0,0 +1,73 @@
+#include <Eigen/Dense>
+#include <complex>
+#include <iostream>
+#include <random>
+#include <vector>
+
+#ifndef NETKET_ACTIVATIONS_HPP
+#define NETKET_ACTIVATIONS_HPP
+
+namespace netket {
+
+class AbstractActivation {
+ public:
+  using VectorType = Eigen::Matrix<std::complex<double>, Eigen::Dynamic, 1>;
+
+  virtual inline void activate(const VectorType &Z, VectorType &A) = 0;
```

Comment:
The rest of netket uses `CamelCase` for member functions. Is there a reason to do it differently here?

Reply:
Good point